lib/ansible/modules/cloud/amazon/dynamodb.py

```diff
@@ -0,0 +1,377 @@
+  dynamodb:
+    profile: pre
+    table: narcos
+    action: get
+    filter_expression: '#p = :project'
+    expression_attribute_values: {":project": {"S": "narcos"}}
+    expression_attribute_names: {"#p" : "project"}
+
+# Creates a single item in 'narcos' DynamoDB table where column 'bank' equals 'hsbc' string,
+# column 'quantity' equals '1000' numeric and column 'person' equals 'ochoa' string
+- name: Creates a new record
+  dynamodb:
+    table: narcos
+    action: put
+    item: {"bank": {"S": "hsbc"},"quantity": {"N": "1000"},"person": {"S": "ochoa"}}
+
+# Updates the 'quantity' attibute value from a single item.
+- name: Updates arribute 'number'
+  dynamodb:
+    table: narcos
+    action: update
+    primary_key: {"bank": {"S": "hsbc"}}
+    update_expression: 'SET quantity = :number'
+    expression_attribute_values: {":number": {"N": "2000"}}
+
+# Updates the 'status' (dynamodb protected keyword) attibute value from a single item.
+- name: Updates arribute 'status'
+  dynamodb:
+    table: narcos
+    action: update
+    primary_key: {"bank": {"S": "hsbc"}}
+    update_expression: 'SET #s = :number'
+    expression_attribute_values: {":number": {"N": "2000"}}
+    expression_attribute_names: {"#s" : "status"}
+
+# Deletes the 'person' attibute value from a single item. The table has a composite
+# primary key. 'bank' is the primary key and 'quantity' is the sort key.
+- name: Deletes arribute 'person'
+  dynamodb:
+    table: narcos
+    action: update
+    primary_key: {"bank": {"S": "hsbc"}, "quantity": {"N": "1000"}}
+    update_expression: 'REMOVE person'
+
+# Connects with awscli profile 'development' and
+# deletes a single item from 'releases' DynamoDB table
+# where the column 'project' equals string 'potatoes' (also primary key)
+# and column 'version' equals numeric '123456'
+- name: Delete a single item
+  dynamodb:
+    profile: development
+    table: releases
+    action: delete
+    primary_key: {"project": {"S": "potatoes"}}
+    condition_expression: version = :number
+    expression_attribute_values: {":number": {"N": "123456"}}
+'''
+
+RETURN = '''
+returned_items:
+    description: item when you peform a 'get' action
+    returned: success
+    type: list
+    sample:
+        [
+            {
+                "bank": {
+                    "s": "hsbc"
+                },
+                "quantity": {
+                    "n": "1000"
+                },
+                "person": {
+                    "s": "ochoa"
+                }
+            }
+        ]
+scanned_count:
+    description: the number of total items evaluated when you peform a 'get' action
+    returned: success
+    type: int
+    sample: 9589
+consumed_capacity:
+    description: the capacity units when you peform a 'get' action
+    returned: success
+    type: dict
+count:
+    description: the number of items in the response when you perform a 'get' action
+    returned: success
+    type: int
+    sample: 13
+response_metadata:
```

Comment:
not really i guess but since guidelines were saying that its okay i did so 

https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/dev_guide/platforms/aws_guidelines.rst#returning-values